addon/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as ContainerQuery } from './components/container-query';
+export { default as aspectRatio } from './helpers/cq-aspect-ratio';
+export { default as height } from './helpers/cq-height';
+export { default as width } from './helpers/cq-width';
```

Comment:
I would like to add strict-mode tests for these, but I feel like adding ember-template-imports may require a breaking change to the addon.

A minimal test could assert that these aren't undefined